sentinel-adapter/sentinel-zuul-adapter/src/test/java/com/alibaba/csp/sentinel/adapter/gateway/zuul/route/SentinelZuulRouteTest.java

```diff
@@ -0,0 +1,52 @@
+package com.alibaba.csp.sentinel.adapter.gateway.zuul.route;
+
+import com.alibaba.csp.sentinel.adapter.gateway.zuul.api.route.PrefixRoutePathMatcher;
+import com.alibaba.csp.sentinel.adapter.gateway.zuul.api.route.RegexRoutePathMatcher;
+import com.netflix.zuul.context.RequestContext;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.mock.web.MockHttpServletRequest;
+
+import static com.alibaba.csp.sentinel.adapter.gateway.zuul.constants.ZuulConstant.SERVICE_ID_KEY;
+
+/**
+ * @author: jiangzian
+ **/
+public class SentinelZuulRouteTest {
+
+    private final String SERVICE_ID = "servicea";
+
+    private final String SERVER_NAME = "www.example.com";
+    private final String REQUEST_URI = "/servicea/test.jsp";
+    private final String QUERY_STRING = "param1=value1&param";
+
+    private RequestContext requestContext = new RequestContext();
+
+
+    @Before
+    public void setUp() {
+        MockHttpServletRequest request = new MockHttpServletRequest();
+        request.setServerName(SERVER_NAME);
+        request.setRequestURI(REQUEST_URI);
+        request.setQueryString(QUERY_STRING);
+        requestContext.set(SERVICE_ID_KEY, SERVICE_ID);
+        requestContext.setRequest(request);
+        RequestContext.testSetCurrentContext(requestContext);
+    }
+
+    @Test
+    public void testPrefixRoutePathMatche() {
+        PrefixRoutePathMatcher prefixRoutePathMatcher = new PrefixRoutePathMatcher("/servicea/????.jsp");
+        Assert.assertTrue(prefixRoutePathMatcher.test(requestContext));
+    }
+
+    @Test
+    public void testRegexRoutePathMatcher() {
+        RegexRoutePathMatcher regexRoutePathMatcher = new RegexRoutePathMatcher("/servicea[^\\s]+.jsp");
```

Comment:
The  pattern `/service[^\s]+.jsp` seems lack of considered.

`\.jsp` should be better than `.jsp`.
And `[^\s]` which indicates non-space subsequence i don't quite understand here because no space is allowed in URI as we know.

Talk to unit test it's better to have both positive and negative cases. So if you familiar with this could you improve it? Otherwise we can improve it later.

Reply:
Thank you very much for your Suggestions, but these are my lack of consideration. Now I have resubmitted them

Reply:
@jasonjoo2010 Do you have time to check my logic